README.md

```diff
@@ -30,6 +30,13 @@ python3 -m venv env
 source ./env/bin/activate
 ```
 
+or (if you have virtualenvwrapper)
+
+```
+mkvirtualenv env
```

Comment:
```suggestion
mkvirtualenv -p 3.9 env
```
Otherwise it will pick the default python version on the host running the command